boards/arm/nrf52833_pca10100/doc/index.rst

```diff
@@ -0,0 +1,221 @@
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+nRF52833 PDK has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf52833_pca10100 board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF52833 PDK website`_ and `Nordic Semiconductor Documentation library`_
+for a complete list of nRF52833 Development Kit board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (green) = P0.14
+* LED3 (green) = P0.15
+* LED4 (green) = P0.16
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P0.11
+* BUTTON2 = SW2 = P0.12
+* BUTTON3 = SW3 = P0.24
+* BUTTON4 = SW4 = P0.25
+* BOOT = SW5 = boot/reset
+
+Programming and Debugging
+*************************
+
+Applications for the ``nrf52833_pca10100`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
```

Comment:
why not? you enabled jlink, pyocd, and openocd runners

Reply:
Agree, this looks like a copy/paste error